sections/main-product.liquid

```diff
@@ -281,6 +291,36 @@
                     </span>
                   {%- endif -%}
                 </div>
+                {%- if product.quantity_price_breaks_configured? -%}
+                  <volume-pricing class="parent-display" id="Volume-{{ section.id }}">
+                    <span class="caption-large">{{ 'products.product.volume_pricing.title' | t }}</span>
+                    <ul class="list-unstyled">
+                      <li>
+                        <span>{{ 'products.product.volume_pricing.minimum' | t: quantity: product.selected_or_first_available_variant.quantity_rule.min }}</span>
+                        <span>{{ product.selected_or_first_available_variant.price | money_with_currency }}/ea</span>
+                      </li>
+                      {%- for price_break in product.selected_or_first_available_variant.quantity_price_breaks -%}
+                        <li class="{%- if forloop.index >= 3 -%}show-more-item hidden{%- endif -%}">
+                          <span>{{ price_break.minimum_quantity }}<span aria-hidden="true">+</span></span>
+                          <span>{{ price_break.price | money_with_currency }}/ea</span>
+                        </li>
+                      {%- endfor -%}
+                    </ul>
```

Comment:
```suggestion
                    <ul class="list-unstyled no-js-hidden">
                      <li>
                        <span>{{ product.selected_or_first_available_variant.quantity_rule.min }}+</span>
                        <span>{{ product.selected_or_first_available_variant.price | money_with_currency }}/ea</span>
                      </li>
                      {%- for price_break in product.selected_or_first_available_variant.quantity_price_breaks -%}
                        <li class="{%- if forloop.index >= 3 -%}show-more-item hidden{%- endif -%}">
                          <span>{{ price_break.minimum_quantity }}<span aria-hidden="true">+</span></span>
                          <span>{{ price_break.price | money_with_currency }}/ea</span>
                        </li>
                      {%- endfor -%}
                    </ul>
                    <noscript>
                      <ul class="list-unstyled">
                        <li>
                          <span>{{ product.selected_or_first_available_variant.quantity_rule.min }}+</span>
                          <span>{{ product.selected_or_first_available_variant.price | money_with_currency }}/ea</span>
                        </li>
                        {%- for price_break in product.selected_or_first_available_variant.quantity_price_breaks -%}
                          <li>
                            <span>{{ price_break.minimum_quantity }}<span aria-hidden="true">+</span></span>
                            <span>{{ price_break.price | money_with_currency }}/ea</span>
                          </li>
                        {%- endfor -%}
                      </ul>
                    </noscript>
```

Reply:
So far you can see the whole list only with JS. How about to add no-JS experience right here in this PR?

Reply:
So far if JS is off you can see the first three lines only. Eventually we'll need to add it, since it's a minor adjustment I would do in this PR, but it's up to.

Reply:
Perhaps we can add a `refresh` button when JS if off? 🤔

Reply:
oh, it's my bad actually, I didn't account for that 😅. In my opinion, it's again time to ask ourself if we need this no-js here at all?